### DISCUSSION TITLE
Only a small amount of traffic arrived at mirrored local process

### DISCUSSION THREAD

**Discussion by sunng87** [metalbear-co/mirrord#1315] [Q&A] [answered]
Hello mirrord! I found the idea of mirrord is quite interesting and trying it out for our team. 

I'm not sure if it's a bug or some misconfiguration. In my testing setup, I have a deployment with one pod. When using `mirrord exec -t ...`, only a small amount of traffic arrived at my local process, when using the default mirror mode. However, by switching to `--steal` mode, it works fine that all traffic are handled by my local process.

I'm using 3.34 from archlinux AUR. I just noticed that there is a newer version released and will test it again.

**Comment by aviramha** [comment, marked as answer]
Hey!
Thanks for the feedback. Currently the mirror feature works only for **new** tcp sessions (as it works on TCP level). I assume that the "small amount" is actually new sessions that started since you started mirroring.
We do have plans to make it more robust (i.e mirror ongoing sessions) but not in the next weeks.
Edit:
Tracking issue https://github.com/metalbear-co/mirrord/issues/1317

**Comment by sunng87** [comment]
That makes sense! It would be nice to mention this limitation in docs.

By the way, I was not able to join the discord server as it says `Unable to accept invitation`. You might want to check if the invitation is expired or invalid.

**Reply by aviramha** [reply]
That's weird.. just checked and it works.
Can you try this one?
https://discord.gg/RTYugKV23P

**Reply by sunng87** [reply]
Thank you! The new link works.

**Comment by aviramha** [comment]
Let us know if you want us to add support in this issue https://github.com/metalbear-co/mirrord/issues/1317
